test/interface_adapter/generate_random_meal/GenerateRandomMealControllerTest.java

```diff
@@ -0,0 +1,36 @@
+package interface_adapter.generate_random_meal;
+
+import app.custom_data.RandomRecipesList;
+import org.junit.Test;
+import use_case.generate_meal_by_id.GenerateMealByIDInputBoundary;
+import use_case.generate_meal_by_id.GenerateMealByIDInputData;
+
+import java.util.Random;
+
+import static org.junit.Assert.*;
+
+public class GenerateRandomMealControllerTest {
+    @Test
+    public void execute() {
+        GenerateRandomMealController generateRandomMealController = new GenerateRandomMealController(
+                new GenerateMealByIDInputBoundary() {
+                    @Override
+                    public void execute(GenerateMealByIDInputData inputData) {
+                        boolean inRecipeList = false;
+                        String[] recipes = RandomRecipesList.randomRecipesList;
+                        String selectedRecipe = inputData.getMealID();
+
+                        for (String recipe : recipes) {
+                            if (recipe.equals(selectedRecipe)) {
+                                inRecipeList = true;
+                                break;
+                            }
+                        }
+
+                        assertTrue(inRecipeList);
```

Comment:
Good test to see if a random selected recipe is in the `randomRecipesList` arraylist. I'm not sure if this is necessary, but do you think the test would be more comprehensive if we test if, for example, 100 randomly selected recipes belongs to the `randomRecipesList` arraylist? Due to the fact that the recipe index is randomly generated, testing for one random recipe may be due to luck.

Reply:
Yes. Makes sense I will do that.